src/einsteinpy/symbolic/christoffel.py

```diff
@@ -1,12 +1,13 @@
 import numpy as np
 import sympy
 
-from einsteinpy.symbolic.tensor import Tensor, _change_config
+from einsteinpy.symbolic.tensor import GRTensor, _change_config
 
 
-class ChristoffelSymbols(Tensor):
```

Comment:
What will be the future of Tensor?

Reply:
`Tensor` would refer to any tensor while this new one can solely be used to relativity purpose.
Don't know may deprecate `Tensor` with 0.3.0. Let's see